dts/bindings/ethernet/zephyr,cdc-ecm-ethernet.yaml

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2023 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: USB CDC ECM virtual ethernet controller
+
+compatible: "zephyr,cdc-ecm-ethernet"
+
+include: ethernet.yaml
+
+properties:
+  mac-address:
+    type: string
```

Comment:
I think MAC address should be `required: true` property.

Reply:
However, not sure how feasible it is to have mac address as a device tree property as these things are usually stored in some configuration memory.

Reply:
Maybe `remote-mac-address:` is a better property name.